src/main/java/lotto/service/LottoServiceImpl.java

```diff
@@ -0,0 +1,77 @@
+package lotto.service;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import lotto.model.Lotto;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/* 실제 LottoService 기능 구현*/
+public class LottoServiceImpl implements LottoService {
+    @Override
+    public List<List<Integer>> generateLottoNumbers(int numberOfLottos) {
+        List<List<Integer>> lottoNumbersList = new ArrayList<>();
+        for (int i = 0; i < numberOfLottos; i++) {
+            List<Integer> lottoNumbers = Randoms.pickUniqueNumbersInRange(1, 45, 6);
+            lottoNumbers.sort(Integer::compareTo);
+            lottoNumbersList.add(lottoNumbers);
```

Comment:
불변 객체를 sorted 하고 있어서 문제가 발생하고 있음
실제로는 pickUniqueNumbersInRange로 값이 만들어지지만 테스트 상황에서는 임의 값을 넣어주고 있기 때문
```suggestion
     List<Integer> mutableNumbers = new ArrayList<>(lottoNumbers);
            mutableNumbers.sort(Integer::compareTo);
            lottoNumbersList.add(mutableNumbers);
```
불변 객체를 가변 객체로 만들어 주면 통과~